formatters/jdt-core/build.properties

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright 2010-2016. All work is copyrighted to their respective
```

Comment:
I respect your point of view and thank you for all the detailed information.  The intended audience of the license changes you refer are for "Apache committers and PMC members".  We are neither.

I do feel some clarity is necessary.  The license header we use is a combination of two things.

* Our Personal notice and copyright
```
Copyright ${license.git.copyrightYears}. All work is copyrighted to their respective
author(s), unless otherwise stated.
```
* The license we provide our project under (Apache license)

The intent of the first (ours) is to designate when we started the project and when the file was last modified.  This is entirely automated per license plugin.

Based on your statements and what I read [here](http://www.apache.org/legal/src-headers.html#notice), the switch would essentially make it appear as if we were actually writing code under ASF which we are not.  We are simply using their license.  Given we are not the intended audience of this guidence and what this intent is, I feel we are fine.

However, I did add typical NOTICE and LICENSE then removed the old license.txt.

Is there something else I'm missing?